### DISCUSSION TITLE
使用dev分支的.cls后是否需要更新main.tex

### DISCUSSION THREAD

**Discussion by ymzang** [obster-y/XJTU-thesis#83] [General]
作者你好，
我最开始写作论文的时候，采用的是研究生院的版本，即master分支。
但由于字符与加粗问题，改用dev分支（20220530），则是否只需要更新.cls文件即可？
其他部分内容（尤其是main.tex）需要变动吗？

**Comment by obster-y** [comment]
emm 只要编译结果正常，改不改不影响。
我是记不太清楚了，main.tex 里变动的内容应该主要就是参考文献了吧。
